### PR TITLE
RST-445 Gatling setup

### PR DESCRIPTION
**Pull request by AlexaVB**
Setup [Gatling ](http://gatling.io/)

### CODE REVIEW

File: performance-tests/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+results/*
+.DS_Store
+.idea/
+target/
+gatling-charts-highcharts-bundle-2.2.5/
```
Comment (leoapost) [inline performance-tests/.gitignore]: I think we can remove this line

File: performance-tests/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,35 @@
+FROM java:8-jdk-alpine
+
+MAINTAINER alexa ballantine <alexa.ballantine@gmail.com>
+
+# working directory for gatling
+WORKDIR /opt
+
+# gating version
+ENV GATLING_VERSION 2.2.2
```
Comment (leoapost) [inline performance-tests/Dockerfile]: How about we use the latest version of Gatling? Should be `v2.2.5`.

File: performance-tests/.gitignore (hunk 1 above)
Comment (AlexaVB) [inline performance-tests/.gitignore]: done

File: performance-tests/Dockerfile (hunk 2 above)
Comment (AlexaVB) [inline performance-tests/Dockerfile]: done

**Pull request merged**